js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/AddIndexForm.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Stack } from "@chakra-ui/react";
+import React, { useState } from "react";
+import SingleSelect from "../../../../components/select/SingleSelect";
+import { useCollectionIndicesContext } from "../CollectionIndicesContext";
+import { FulltextIndexForm } from "./FulltextIndexForm";
+import { GeoIndexForm } from "./GeoIndexForm";
+import { InfoTooltip } from "./InfoTooltip";
+import { PersistentIndexForm } from "./PersistentIndexForm";
+import { TTLIndexForm } from "./TTLIndexForm";
+import { ZKDIndexForm } from "./ZKDIndexForm";
+
+export const AddIndexForm = ({ onClose }: { onClose: () => void }) => {
+  const { indexTypeOptions } = useCollectionIndicesContext();
+  const [indexType, setIndexType] = useState(indexTypeOptions?.[0].value || "");
+  let tooltipText = "Type of index to create.";
+  if (!indexTypeOptions?.find(option => option.value === "hash")) {
+    tooltipText = `${tooltipText} Please note that for the RocksDB engine the index types "hash", "skiplist' and "persistent" are identical, so that they are not offered seperately here.`;
```

Comment:
Typo

```suggestion
    tooltipText = `${tooltipText} Please note that for the RocksDB engine the index types "hash", "skiplist" and "persistent" are identical, so that they are not offered separately here.`;
```